java/server/src/org/openqa/selenium/grid/distributor/model/Host.java

```diff
@@ -163,27 +165,34 @@ public UUID getId() {
     return new DistributorStatus.NodeSummary(
         nodeId,
         uri,
-        getHostStatus() == UP,
+        node.isDraining(),
```

Comment:
This has been changed to expect `Status` in another part of the patch so won't compile.